### PR TITLE
basic botkit implementation of https://github.com/cdbhnd/botkit-viber

### PR DESCRIPTION
**Pull request by maaraanas**
need alot of cleanup and also merge it into one piece, but basic use is ok here)

### CODE REVIEW

File: examples/viber_bot.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    default: true,
+                    callback: function (response, convo) {
+                        convo.gotoThread('default');
+                    },
+                }
+            ], {}, 'default');
+
+            convo.activate();
+        });
+    });
```
Comment (mokeddes) [inline examples/viber_bot.js]: Return to the line (Newline at the end of file )

Comment (mokeddes) [reply examples/viber_bot.js]: Recommanded to use arrow function

File: lib/ViberMessageTypes.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        'media': mediaUrl,
+    };
+
+    if (thumbnailUrl) { message[thumbnail] = thumbnailUrl; };
+
+    return message;
+}
+
+function VideoMessage(mediaUrl, size, duration, thumbnail) {
+    let message = {
```
Comment (mokeddes) [inline lib/ViberMessageTypes.js]: Why not factor Message ? Like This:
let message = {
'receiver': '',
        'min_api_version': 1,
        'sender': {
            'name': '',
            'avatar': ''
        }
}

And you will add some proprieties for the message in all function

File: package.json [hunk 3]
```diff
@@ -24,6 +24,7 @@
     "mustache": "^2.3.0",
     "promise": "^8.0.0",
     "request": "^2.81.0",
+    "request-promise": "^4.2.2",
```
Comment (mokeddes) [inline package.json]: Do you have see node-fetch or axios ?

File: lib/ViberMessageTypes.js (hunk 2 above)
Comment (maaraanas) [reply lib/ViberMessageTypes.js]: thanks! i'll fix this asap. Had no time in past few days at all

**Pull request closed without merging**